Makefile.dep

```diff
@@ -2,6 +2,9 @@
 OLD_USEMODULE := $(sort $(USEMODULE))
 OLD_USEPKG := $(sort $(USEPKG))
 
+# include application dependencies
+-include $(APPDIR)/Makefile.dep
```

Comment:
If you change just change this line to something like
```suggestion
-include $(APPDIR)/Makefile.board_dep
```

It *still* works as proposed, but without the confusion about where to put the dependencies / module inclusions of an application.

Reply:
I thought the comment `#Put here board specific dependencies` I added in each application Makefile.dep would avoid confusion. The other dependencies remain the main application Makefile.

Reply:
Some users are notoriously good in overlooking comments ;-).